packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -294,7 +294,7 @@ const ModalManager = () => {
             {is_jurisdiction_modal_visible && <JurisdictionModal openPasswordModal={openRealPasswordModal} />}
             {should_show_cfd_password_modal && <CFDPasswordModal platform={platform} />}
             {is_cfd_verification_modal_visible && <CFDDbviOnBoarding />}
-            {is_cfd_reset_password_modal_enabled && <CFDResetPasswordModal platform={platform} />}
+            <CFDResetPasswordModal platform={platform} /> {/* a new condition for this hotfix needs to be found */}
```

Comment:
just checking why do we need this change now?

Reply:
The condition that was used here does not allow the modal to pop up. As it is a hotfix and using different conditions did not work, we made it as it was before in order to work

Reply:
A comment is added so that we find the proper condition for it and since we need to unblock the tag, this is the quickest solution

Reply:
the component itself already has the check:

```JavaScript
    const getIsListFetched = () => {
        return Object.keys(current_list).length !== 0;
    };

    const is_invalid_investor_token = !getIsListFetched() && localStorage.getItem('cfd_reset_password_code');

    return (
        <Modal is_open={is_cfd_reset_password_modal_enabled && !is_invalid_investor_token} >
```

so having the check 
`{is_cfd_reset_password_modal_enabled && <CFDResetPasswordModal platform={platform} />}`  \
is redundant

we can also move the conditions out of the component, into the modal manager, but this will require passing more props down the `<CFDResetPasswordModal />` component